docs/wiki/deployment/configuration.md

```diff
@@ -188,56 +205,197 @@ And the same configuration file from the OS X example is appropriate.
 
 ## Query Packs
 
-Configuration supports sets, called packs, of queries that help define your schedule. Packs are distributed with osquery and labeled based on broad categories of information and visibility. For example, a "compliance" pack will include queries that check for changes in locked down operating system features and user settings. A "vulnerability management" pack may perform general asset management queries that build event logs around package and software install changes.
+Configuration supports sets, called packs, of queries that help define your
```

Comment:
Did you use an auto formatter for these paragraphs, or something that enforced hard-wrapping? Manually limiting prose to 80 character line widths seems cumbersome for little gain if your editor supports soft-wrapping.